frontend/src/static/js/v_authorship.js

```diff
@@ -107,11 +117,11 @@ window.vAuthorship = {
         file.className = renameValue;
       });
 
-      this.activeFilesCount = isActive ? this.selectedFiles.length : 0;
+      this.isActive = isActive;
     },
 
     updateCount() {
-      this.activeFilesCount = document.getElementsByClassName('file active').length;
+      this.isActive = document.getElementsByClassName('file active').length;
```

Comment:
Shouldn't this line be `this.isActive = document.getElementsByClassName('file active').length > 0;`?

Reply:
don't need. .length is enough alr

Reply:
I know it works, but it would be confusing for future developers if `this.isActive` is sometimes a `Boolean` and other times a `Number`, and the name implies that it is a boolean value also.

Reply:
done